### ISSUE TITLE
Index build grunt task not working Properly

### ISSUE THREAD

**Issue by ashpareek** [ngbp/ngbp#372]
Using this task import files are adding in index.html in wrong order.

**Comment by adamjaffeback** [comment]
+1 
`grunt watch` fails due to: 

``` sh
Running "less:build" (less) task
>> FileError: '../../vendor/bootstrap/less/bootstrap.less' wasn't found in src/less/main.less on line 13, column 1:
>> 12 
>> 13 @import '../../vendor/bootstrap/less/bootstrap.less';
>> 14
Warning: Error compiling src/less/main.less Use --force to continue.

Aborted due to warnings.
```

**Comment by mustela** [comment]
@adam-back did you run `bower install` ?

**Comment by adamjaffeback** [comment]
Yes, I did. I ran through the entire Quick Start. I also confirmed that the
the `bower_components` existed.

On Thu, Apr 2, 2015 at 4:24 PM, Emiliano Jankowski <notifications@github.com

> wrote:
> 
> @adam-back https://github.com/adam-back did you run bower install ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ngbp/ngbp/issues/372#issuecomment-89076450.

## 

Personal Webpage http://www.adam-back.com/
(w/links to GitHub, Linkedin, etc.)

**Comment by adamjaffeback** [comment]
Fixed:
I found when I cloned the repo down, some files were missing, including (but not limited to) `.bowerrc` and `.gitignore`. Therefore, my `bower install` was not going to the correct place.

**Comment by adamjaffeback** [comment]
@ashpareek, check to make sure you have a `.bowerrc` file in the root directory. It should contain:

``` sh
{
  "directory": "vendor",
  "json": "bower.json"
}
```
